etcdctl/command/cluster_health.go

```diff
@@ -44,7 +45,8 @@ func handleClusterHealth(c *cli.Context) {
 		Transport: tr,
 	}
 
-	mi := mustNewMembersAPI(c)
```

Comment:
The environment is a codegangsta/cli.Context, and I don't find the way to set a flag. It looks read-only.